src/components/Settings/SettingsMain/index.tsx

```diff
@@ -427,7 +434,9 @@ const SettingsMain = () => {
                     <span className="mr-2">
                       {intl.formatMessage(messages.hideAvailable)}
                     </span>
-                    <SettingsBadge badgeType="experimental" />
```

Comment:
Hide available should have experimental tag. Shouldn't be removed in this pr as its not related to this